docs/pages/workflow/expo-cli.md

```diff
@@ -5,1083 +5,284 @@ maxHeadingDepth: 4
-  - Dropped aliases for `--dump-assetmap` (`-d`) and `--dump-sourcemap` (`-s`).
-  - Dropped warning about clearing `dist` folder that is shown between runs, this is akin to all other bundler commands across web tooling.
-  - Dropped all merging code because experimental bundle does not support index files so we don't need to support merging index files.
-  - Dropped the quiet flag as this appears to do nothing anymore.
-- Parts of `expo upgrade` and `expo doctor` have been merged into `npx expo install` via the `--fix` and `--check` flags. [PR](https://github.com/expo/expo/pull/17048).
-- Dropped unused generated `.exprc`, `.expo/packager-info.json` files.
-- Dropped all deprecated support for `*.expo.js` files.
-- Reduced usage of `.expo/settings.json` to provide a more sandboxed state between processes.
-
-#### Environment Variables
-
-- Dropped `$XDL_PORT`, `$XDL_HOST`, `$XDL_SCHEME`, `$SERVER_URL` in favor of `$EXPO_STAGING` and `$EXPO_LOCAL` environment variables.
-- Dropped `$EXPO_PACKAGER_HOSTNAME`, `$EXPO_MANIFEST_PROXY_URL` environment variables.
-- Dropped `$EXPO_EDITOR` in favor of `$EDITOR` environment variable.
-- Utilize `$DEBUG=expo:*` more and `$EXPO_DEBUG` less (more control).
+- `EXPO_NO_WEB_SETUP` (**boolean**) prevents the CLI from forcing web dependencies (`react-dom`, `react-native-web`, `@expo/webpack-config`) to be installed before using web functionality. This is useful for cases where you wish to perform non-standard web development.
+- `EXPO_NO_TYPESCRIPT_SETUP` (**boolean**) prevents the CLI from forcing TypeScript to be configured on `npx expo start`. [Learn more](/guides/typescript/).
+- `DEBUG=expo:*` (**string**) enables debug logs for the CLI, you can configure this using the [`debug` convention](https://github.com/debug-js/debug#conventions).
+- `EXPO_DEBUG` (**boolean**) an alias for `DEBUG=expo:*`.
+- `EXPO_PROFILE` (**boolean**) enable profiling stats for the CLI, this does not profile your application.
+- `EXPO_NO_CACHE` (**boolean**) disable all global caching. By default Expo config JSON schemas, Expo Go binaries for simulators and emulators, and project templates are cached in the computer's global `.expo` folder.
+- `CI` (**boolean**) when enabled, the CLI will disable interactive functionality, skip optional prompts, and fail on non-optional prompts. Example: `CI=1 npx expo install --check` will fail if any installed packages are outdated.
+- `EXPO_NO_TELEMETRY` (**boolean**) disables anonymous usage collection.
+- `EXPO_NO_GIT_STATUS` (**boolean**) skips warning about git status during potentially dangerous actions like `npx expo prebuild --clean`.
+- `EXPO_ENABLE_INTERSTITIAL_PAGE` (**boolean**) enables the experimental "interstitial page" for selecting an app, that is shown when a user has `expo-dev-client` installed, and starts the project with `expo start` instead of `expo start --dev-client`.
```

Comment:
```suggestion
- `EXPO_ENABLE_INTERSTITIAL_PAGE` (**boolean**) enables the experimental "interstitial page" for selecting an app, that shows when a user has `expo-dev-client` installed, and starts the project with `expo start` instead of `expo start --dev-client`.
```